tests/Integration/Behaviour/Features/Context/Domain/Product/CommonProductFeatureContext.php

```diff
@@ -33,12 +33,24 @@
 use PrestaShop\PrestaShop\Core\Domain\Product\Exception\ProductConstraintException;
 use PrestaShop\PrestaShop\Core\Domain\Product\Exception\ProductException;
 use PrestaShop\PrestaShop\Core\Domain\Product\Exception\ProductNotFoundException;
+use PrestaShop\PrestaShop\Core\String\CharacterCleaner;
 use RuntimeException;
 use Tests\Integration\Behaviour\Features\Context\Util\CombinationDetails;
 use Tests\Integration\Behaviour\Features\Context\Util\ProductCombinationFactory;
 
 class CommonProductFeatureContext extends AbstractProductFeatureContext
 {
+    /**
+     * @var CharacterCleaner
+     */
+    private $characterCleaner;
+
+    /** @BeforeScenario */
```

Comment:
```suggestion
    /** @BeforeFeature */
```

Unless it needs to be refreshed at each scenario?

Reply:
actually it wasn't even used. And even if it was, I could add a constructor instead of this. 
For now I just removed it (my intentions was to use it for html characters decoding, but it wasn't made for that so it ended up unused)